docker-compose.yml

```diff
@@ -77,6 +77,29 @@ services:
       - '1025:1025'
       - '8025:8025'
     image: mailhog/mailhog
+  # The Locust engine is used for load testing purposes.  
```

Comment:
This should probably go in a separate `docker-compose.loadtest.yml` or similar. I wouldn't expect to kick off a load test by running `docker-compose up`

Reply:
Note, this does not kick off a load test. You have to manually kick one off by going to the Locust container's URL and interacting with its frontend. This simply makes the testing tool available in the development stack.

Reply:
![image](https://user-images.githubusercontent.com/93527746/157347583-adefa24d-d750-45f8-ab67-261562f66be1.png)

Reply:
I guess I don't agree that it's part of the application stack. I'd prefer to not have extra containers consuming resources unless I'm running a load test. Also, having it part of a separate file makes it a little more straightforward to run in CI.

Reply:
This should now be addressed. @alismx, I think this should address your concerns, too...please let me know if there is anything else either of you would like to see added! I have also updated the documentation to clarify what is needed to actually engage a load test.

Reply:
Thank you! I honestly did not know that was possible. I've updated this PR with that change!